ui/app/pages/send/send.component.js

```diff
@@ -160,13 +162,18 @@ export default class SendTransactionScreen extends Component {
     let scannedAddress
     if (qrCodeData) {
       if (qrCodeData.type === 'address') {
-        scannedAddress = qrCodeData.values.address.toLowerCase()
```

Comment:
This line still seems important :thinking: 
Where is `scannedAddress` set otherwise?

Reply:
I deleted too many lines... <_< I restored the most important line in the QR code functionality. :eyes: